client/styles/components/_github-button.scss

```diff
@@ -42,7 +47,11 @@
   flex-direction: row;
   justify-content: center;
   align-items: center;
-  width: #{300 / $base-font-size}rem;
+  width:95%;
```

Comment:
rather than doing a percent width here, i think it makes sense to add a fixed padding or margin.